### PR TITLE
Change venue models for new convention

### PR DESCRIPTION
**Pull request by pulsejet**
The new convention tries to stick to the format of locations used by instiMAP for consistency, which includes using `short_name` for names like VMCC and `name` for Victor Menezes Convention Center. We want to show `short_name` to the user everywhere.

### CODE REVIEW

File: app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Venue.java [hunk 1]
```diff
@@ -16,16 +16,48 @@
...
+    Integer venueGroupId;
+    @ColumnInfo(name = "pixel_x")
+    @SerializedName("pixel_x")
+    Integer venuePixelX;
+    @ColumnInfo(name = "pixel_y")
+    @SerializedName("pixel_y")
+    Integer venuePixelY;
+    @ColumnInfo(name = "reusable")
+    @SerializedName("reusable")
+    Boolean venueReusable;
```
Comment (unstablebrainiac) [inline app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Venue.java]: Shouldn't there be a Venue object in place of all these?

Comment (pulsejet) [reply app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Venue.java]: This is the venue model, right? Didn't really catch that.

Comment (unstablebrainiac) [reply app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Venue.java]: There is, although you'll have to edit it to include short names and other new fields

Comment (pulsejet) [reply app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Venue.java]: Really confused now. Isn't that exactly what I've done?

Comment (unstablebrainiac) [reply app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Venue.java]: Misread, my bad

**Pull request merged**